loop/forms.py

```diff
@@ -72,4 +75,34 @@ def clean_farmer_file(self):
             raise forms.ValidationError("Please select atleast one Cluster or .csv file")
         return farmer_file
 
+class MergeEntityForm(forms.Form):
 
+    choice_set = [('', '----------')]   
+    for key in cnf.models:
+        choice_set.append((key, cnf.models[key]['display_value']))
+    model = forms.ChoiceField(required=True,widget= forms.Select, label='Select Model:', choices=choice_set)
+    merge_file = forms.FileField(required=True, label='Select a .csv file:',)
+    email = forms.EmailField(required=True, label='Enter email address to receive status of merge request:')
+
+
+    def __init__(self, *args, **kwargs):
+        super(MergeEntityForm, self).__init__(*args, **kwargs)
+        self.fields['merge_file'].error_messages = {'required':'Merge file is required'}
+        self.fields['model'].error_messages = {'required':'Select model name'}
+        self.fields['email'].error_messages = {'required': 'Enter email address'}
+        self.fields['merge_file'].help_text = ""
+
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+        try:
+            validate_email(email)
+            return email
+        except ValidationError:
```

Comment:
use "except validate_email.ValidationError: " for catch exception of validate_email.